app/src/main/java/com/bunbeauty/papakarlo/feature/order/screen/orderdetails/OrderProductItemMapper.kt

```diff
@@ -20,7 +20,17 @@ class OrderProductItemMapper(
                 stringUtil.getCostString(oldCost)
             },
             photoLink = stringUtil.getCostString(orderProductItem.photoLink),
-            count = stringUtil.getCountString(orderProductItem.count)
+            count = stringUtil.getCountString(orderProductItem.count),
+            key = "${orderProductItem.uuid} ${
```

Comment:
`"OrderProduct ${orderProductItem.uuid}"` will be enough

Reply:
can we have same products in one order?